README.md

```
@@ -380,15 +428,65 @@ All tables use Row Level Security (RLS). Reads and writes require authenticated
+app/*/*/__tests__/      # Co-located with pages
+lib/__tests__/          # Utilities & data layer (stay centralized)
+```
 
 ---
 
-### Unit Tests — `payroll.test.ts`
+### Test Coverage Summary (April 2026)
+
+| Area | Test File | Tests | Passing | Skipped |
+|------|---------|-------|--------|---------|
+| **lib/** | | | | | |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 5; Actual: 6; Too many cells, extra data will be missing

(MD056, table-column-count)